tiny_hippo_run.py

```diff
@@ -79,22 +59,23 @@ def packet_parse(packet: Packet):
             rule(packet)
         except Exception as e:
             # TODO: refine so a specific error message can be logged
-            run_config.log_event.info('Exception raised: ' + str(e))
+            run_config.log_event.info('Exception raised in a privacy rule check: ' + str(e))
     # For each triggered signature generate an alert for the user
     try:
         triggered_rules = signature_detector.check_signatures(packet)
         if len(triggered_rules) > 0:
             for triggered_rule in triggered_rules:
-                is_dst = packet[Ether].src in mac_addrs
-                alert_object = Alert(packet, triggered_rule.msg, ALERT_TYPE.IDS, SEVERITY.ALERT, is_dst)
+                is_dst = packet[Ether].src in DeviceInformation.get_mac_addresses()
```

Comment:
Use the global value here, it will reduce the amount of calls we make to DeviceInformation

Reply:
I removed the global value, we want to be querying the database for mac addresses to prevent stale data from being used.